game/script-ch30.rpy

```diff
@@ -124,27 +173,82 @@ label ch30_loop:
 
 label ch30_wait:
     window hide
-    $ renpy.pause(delay=5.0, hard=True)
+    python:
+        import random
+
+        if (random.randint(1, 10000) == 1):
+            jn_stickers.stickerWindowPeekUp(at_right=random.choice([True, False]))
+
+        renpy.pause(delay=5.0, hard=True)
+    
     jump ch30_loop
 
 #Other labels
 label call_next_topic(show_natsuki=True):
     if show_natsuki:
-        show natsuki at jn_center
+        show natsuki idle at jn_center zorder JN_NATSUKI_ZORDER
 
     if persistent._event_list:
         $ _topic = persistent._event_list.pop(0)
 
         if renpy.has_label(_topic):
             # Notify if the window isn't currently active
-            if (persistent.jn_notify_conversations
+            if (persistent._jn_notify_conversations
                 and jn_utils.get_current_session_length().total_seconds() > 60
-                and not jn_activity.get_jn_window_active()):
+                and not jn_activity.getJNWindowActive()
+                and not _topic in ["random_music_change", "weather_change"]):
+                    
                     play audio notification
-                    $ jn_activity.taskbar_flash()
+                    python:
+                        jn_activity.taskbarFlash()
+
+                        if Natsuki.isNormal(higher=True):
+                            if Natsuki.isEnamored(higher=True):
+                                notify_message = random.choice([
```

Comment:
Messages here are hardcoded, imo would be better to have them as constants.

Reply:
Sure, I can do that